### PR TITLE
fix(translations): fix Spanish translation for "breadcrumbs"

### PR DESCRIPTION
**Pull request by TheMineWay**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Improve spanish translations to keep the software understandable for all.
The previous translation was correct but it was too literal (spanish people would not understand what "Migas de pan" is in a website. Even developers). By using "Rastro de navegación" people would understand better the concept.
This does not fix any issue nor bug.

## Test Plan

Spanish speakers can verify that the new translation is better to understand.

### Test links

Deploy preview: https://deploy-preview-10413--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/docusaurus-theme-translations/locales/es/theme-common.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
   "theme.contentVisibility.unlistedBanner.title": "Página sin clasificar",
   "theme.docs.DocCard.categoryDescription.plurals": "1 artículo|{count} artículos",
   "theme.docs.breadcrumbs.home": "Página de Inicio",
-  "theme.docs.breadcrumbs.navAriaLabel": "Migas de pan",
+  "theme.docs.breadcrumbs.navAriaLabel": "Rastro de navegación",
```
Comment (slorber) [inline packages/docusaurus-theme-translations/locales/es/theme-common.json:52]: Are you sure?

This has been submitted recently in https://github.com/facebook/docusaurus/pull/10360 and I double checked and apparently `Migas de pan` is more common than `Rastro de navegación` (but I'm not a Spanish native speaker so can't be 100% sure)

Can you say something @sergioalmela ?

**Comment by slorber** [comment]
Thanks

Can you back your claim with a link to an authoritative source (like Google?) favoring one term over the other?

Because from my perspective the tools I can use disagree with you

CleanShot 2024-08-19 at 14 21 09@2x

File: packages/docusaurus-theme-translations/locales/es/theme-common.json (hunk 1 above)
Comment (slorber) [reply packages/docusaurus-theme-translations/locales/es/theme-common.json:52]: I can't comment on that, unfortunately, and will let Spanish users settle.

The only thing I see is that Google uses another term: Ruta de navegación/exploración

https://developers.google.com/search/docs/appearance/structured-data/breadcrumb?hl=es

**Comment by sergioalmela** [comment]
Maybe I'm very used to say migas de pan because we said that in my company, but to be honest maybe what you said is more understandable to non-technical users.

**Comment by slorber** [comment]
Thanks @sergioalmela , so you agree merging this PR would be fine?

**Comment by sergioalmela** [comment]
> Thanks @sergioalmela , so you agree merging this PR would be fine?

Yes

**Comment by slorber** [comment]
Thanks 👍

**Pull request merged**